alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureEmailGroupsRestApiIT.kt

```diff
@@ -60,7 +60,10 @@ class SecureEmailGroupsRestApiIT : AlertingRestTestCase() {
 
         if (userClient == null) {
             createUser(user, user, arrayOf())
-            userClient = SecureRestClientBuilder(clusterHosts.toTypedArray(), isHttps(), user, user).setSocketTimeout(60000).build()
+            userClient = SecureRestClientBuilder(clusterHosts.toTypedArray(), isHttps(), user, user)
+                .setSocketTimeout(60000)
+                .setConnectionRequestTimeout(180000)
```

Comment:
What is the reason of this request timeout?

Reply:
This is the related [PR](https://github.com/opensearch-project/notifications/pull/583) to prevent DeadlineTimeoutException